client/src/main/java/tw/waterball/survive/io/client/view/StandardGameView.java

```diff
@@ -1,5 +1,67 @@
-package tw.waterball.survive.io.client.view;
+package tw.waterball.survive.io.client.view.viewImpl;
+
+import tw.waterball.survive.io.client.sprite.Role.Role;
+import tw.waterball.survive.io.client.sprite.Sprite;
+import tw.waterball.survive.io.client.sprite.Weapon.WeaponSprite;
+import tw.waterball.survive.io.client.view.GameView;
+
+import javax.sound.sampled.*;
+import java.io.File;
+import java.io.IOException;
+import java.text.NumberFormat;
 
 public class StandardGameView extends GameView {
 
+    @Override
+    public void launch() {
+
+    }
+
+    @Override
+    public void onRender(Sprite[] sprites) {
+
+    }
+
+    @Override
+    public void onAttack(WeaponSprite weaponSprite, Role role) {
+        playAudioEffect(weaponSprite.getSoundPath());
+    }
+
+    private void playAudioEffect(String fileName) {
```

Comment:
You should create an util class under a `utils/` package
and encapsulate these Audio stuff into that that util class.

If you don't understand, feel free to ask @a8901053  in the group.